entrypoint.sh

```diff
@@ -52,14 +52,19 @@ fi
 
 RESPONSE=$(
   curl \
-    --fail \
     --silent \
     -X POST \
     -H "Authorization: Bearer ${BUILDKITE_API_ACCESS_TOKEN}" \
     "https://api.buildkite.com/v2/organizations/${ORG_SLUG}/pipelines/${PIPELINE_SLUG}/builds" \
     -d "$JSON"
 )
 
+if [[ $(jq .id <<< "$RESPONSE") == null ]]; then
+  echo -n "Buildkite API call failed: "
+  jq .message <<< "$RESPONSE"
```

Comment:
I think we keep this consistent with the rest of the file and switch it to:
```suggestion
if [[ $(echo "$RESPONSE" | jq .id) == null ]]; then
  echo -n "Buildkite API call failed: "
  echo "$RESPONSE" | jq --raw-output ".message"
```